docs/reference/components/sources/apache_metrics.cue

```diff
@@ -0,0 +1,213 @@
+package metadata
+
+import (
+  "strings"
+)
+
+components: sources: apache_metrics: {
+  title: "Apache HTTPD Metrics"
+  description: strings.ToTitle(classes.function) + " metrics from an Apache HTTPD server."
```

Comment:
Would it be possible for this to be something like `metrics from an Apache HTTPD server` and have `cue` prepend `strings.ToTitle(classes.function)`? I notice we have that repeated in all of the `description`s.

Reply:
Yeah, I ended up dropping the `function` part. I'd like to keep the descriptions as full sentences since it's easier to understand. That was a point of confusion in the last system. We had things like `through_description` and it wasn't clear how that was used in a sentence.